_pytest/mark.py

```diff
@@ -335,3 +339,6 @@ def add_mark(self, mark):
     def __iter__(self):
         """ yield MarkInfo objects each relating to a marking-call. """
         return imap(MarkInfo, self._marks)
+
+
+MARK_GEN = MarkGenerator()
```

Comment:
Actually I was proposing just the variable:

```python
mark_generator = MarkGenerator()
```

But if you are willing, I think "factory" conveys a better name here (that word is also used in the docstring in fact). So perhaps:

```python
mark_factory = MarkFactory()
```